test/e2e/service.go

```diff
@@ -72,9 +72,19 @@ var _ = framework.KubeDescribe("Services", func() {
 	f := framework.NewDefaultFramework("services")
 
 	var c *client.Client
+	var gceController *GCEIngressController
 
 	BeforeEach(func() {
 		c = f.Client
+		By("Initializing gce controller")
+		gceController = &GCEIngressController{ns: f.Namespace.Name, Project: framework.TestContext.CloudConfig.ProjectID, c: c}
+		gceController.init()
+	})
+
+	// Platform specific cleanup
+	AfterEach(func() {
+		By("Cleaning up cloud resources")
+		cleanupGCE(gceController)
```

Comment:
That's fine, I don't think this is a p0 since it's a slow leak and we're not going to block release on it